src/components/List.js

```diff
@@ -146,10 +146,17 @@ const List = ({ token }) => {
           {docs.length === 0 ? (
             <section>
               <p>Your shopping list is currently empty.</p>
-              <a href="/add-item">Add an Item</a>
+              <a className="add-item-link" href="/add-item">
```

Comment:
added same styling and icon for the empty shopping list view

Reply:
👍 Sorry in advance if I missed any little changes in my PR. Definitely not intentional, so I can make any other remaining updates if I missed anything. Thank you @DeeDowns for taking care of these little details that go a long way towards making our app look clean!

Reply:
👍

Reply:
Love it. I really like the hover effect!